### ISSUE TITLE
Add a contributors.md file

### ISSUE THREAD

**Issue by kempsterrrr** [Developer-DAO/web3-starter-kits#6]
As we put some energy behind this initiative is it worth adding a contributors.md file to help folks understand how to get their repo included or contribute to DAO maintain repo's

**Comment by billyjitsu** [comment]
yeah, do we want to put this on here or another contributor repo?

**Comment by kempsterrrr** [comment]
Let's keep it simple and do it in this repo @billyjitsu
